src/main/webapp/customer-center/new-subscription.jsp

```diff
@@ -18,73 +20,153 @@
 <%@include file="../fragments/navbar.jsp" %>
 
 <div class="container-fluid bg-1 text-center">
-    <img src="${pageContext.request.contextPath}/images/logo-big.png" class="img-responsive img-circle" style="display:inline;" width="150" height="150">
+    <img src="${pageContext.request.contextPath}/images/logo-big.png" class="img-responsive img-circle"
+         style="display:inline;" width="150" height="150">
     <h2>Új előfizetés</h2>
 </div>
 
+<script>
+
+    function chooseProduct(id, quantity) {
+        $.post(
+            '${pageContext.request.contextPath}/customer-center/select-product',
+            {id: id, quantity: quantity},
+            (data, status) => {
+
+            }
+        )
+    }
+</script>
+
+<sql:query var="products" dataSource="jdbc/mysql">
+    select id, name, price, description, image_path
+    from product;
+</sql:query>
+
+<sql:query var="frequencies" dataSource="jdbc/mysql">
+    select id, name
+    from frequency;
+</sql:query>
+
 <div class="container container-fluid">
 
-	<sql:query var="products" dataSource="jdbc/mysql">
-		select id, name, price, description, image_path from product;
-	</sql:query>
-			
-	<h2>Termékek kiválasztása</h2>
-			
-	<div class="panel panel-default">
-		<div class="panel-body">
-		
-			<c:forEach items="${products.rows}" var="product">
-			
-				<form action="select-product" method="get">
-					
-					<h3>${product.name}</h3>
-					<p>${product.description}</p>
-					
-					<div class="input-group">
-						<span class="input-group-addon">Mennyiség</span>
-					  <input class="form-control" type="number" name="quantity" value="${sessionScope.selectedProducts.get(product.id)}">
-					  <input name="productId" value="${product.id}" hidden>
-					</div><br>
-					
-					<button type="submit" class="btn btn-lg btn-block bg-1" title="">Termék kiválasztása</button>
-					
-				</form>
-			
-			</c:forEach>
-			
-		</div>
-	</div>
-	
-
-<form action="new-subscription" method="post">
-	
-	<div class="panel panel-default">
-		<div class="panel-body">
-			<div class="input-group">
-				<span class="input-group-addon">Cím</span>
-				<input class="form-control" name="address" required placeholder="1027. Budapest Gipsz Jakab utca 4.">
-			</div><br>
-			
-			<sql:query var="frequencies" dataSource="jdbc/mysql">
-				select id, name from frequency;	
-			</sql:query>
-			
-			<div class="input-group">
-				<span class="input-group-addon">Szállítás gyakorisága</span>
-			  <select class="form-control" name="frequencyId">
-			  	<c:forEach items="${frequencies.rows}" var="freq">
-			    	<option value="${freq.id}">${freq.name}</option>
-			    </c:forEach>
-			  </select>
-			</div><br>
-				
-			<button type="submit" class="btn btn-lg btn-block bg-1" title="">Új előfizetés elküldése</button>
-			</div>
-		</div>
-</form>
 
+    <div class="row">
```

Comment:
Nehéz lenne az egészet elmagyarázni, az ötletet és a kódot innen loptam: https://www.w3schools.com/bootstrap/default.asp
Érdemes nézegetni devtools-szal is.